x-pack/metricbeat/module/autoops_es/cat_shards/data.go

```diff
@@ -88,8 +88,9 @@ func eventsMapping(m *elasticsearch.MetricSet, r mb.ReporterV2, info *utils.Clus
 
 	if err != nil {
 		indexMetadata = map[string]IndexMetadata{}
```

Comment:
Because we are returning `nil` below this, we have no need to assign `indexMetadata`.

```suggestion
```